Update pyproject to be compatible with newer build tools and
package versions found in current Debian

--- core-network.orig/daemon/pyproject.toml
+++ core-network/daemon/pyproject.toml
@@ -7,12 +7,12 @@ license = "BSD-2-Clause"
 repository = "https://github.com/coreemu/core"
 documentation = "https://coreemu.github.io/core/"
 include = [
-    "core/api/grpc/*",
-    "core/configservices/*/templates",
-    "core/constants.py",
-    "core/gui/data/**/*",
+    { path = "core/api/grpc/*", format = ["sdist", "wheel"] },
+    { path = "core/configservices/*/templates", format = ["sdist", "wheel"] },
+    { path = "core/constants.py", format = ["sdist", "wheel"] },
+    { path = "core/gui/data/**/*", format = ["sdist", "wheel"] },
 ]
-exclude = ["core/constants.py.in"]
+exclude = ["core/constants.py.in", "core/.gitignore"]
 
 
 [tool.poetry.dependencies]
@@ -22,7 +22,7 @@ fabric = "2.5.0"
 grpcio = "1.27.2"
 invoke = "1.4.1"
 lxml = "4.6.5"
-mako = "1.1.3"
+mako = "1.3.9"
 netaddr = "0.7.19"
 pillow = "8.3.2"
 protobuf = "3.19.4"
@@ -52,5 +52,5 @@ exclude = ".+_pb2.*.py|doc/|build/|__pyc
 
 [build-system]
 requires = ["poetry>=0.12"]
-build-backend = "poetry.masonry.api"
+build-backend = "poetry.core.masonry.api"
 
